homeassistant/components/solarlog/sensor.py

```diff
@@ -4,48 +4,35 @@
 
 from requests.exceptions import HTTPError, Timeout
 from sunwatcher.solarlog.solarlog import SolarLog
-import voluptuous as vol
 
-from homeassistant.components.sensor import PLATFORM_SCHEMA
-from homeassistant.config_entries import SOURCE_IMPORT
-from homeassistant.const import CONF_HOST, CONF_NAME
-import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_HOST
 from homeassistant.helpers.entity import Entity
 from homeassistant.util import Throttle
 
-from .const import DEFAULT_HOST, DEFAULT_NAME, DOMAIN, SCAN_INTERVAL, SENSOR_TYPES
+from .const import SCAN_INTERVAL, SENSOR_TYPES
 
 _LOGGER = logging.getLogger(__name__)
 
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
```

Comment:
Removing this will break the startup if I'm correct...

Reply:
@frenck I've tested it, but it works fine. If I keep the configuration in `configuration.yaml` I get the deprecated warning after a restart. 

Removing and adding the integration in the UI also works fine, no issues. 

In #42535 the replaced it with 

`CONFIG_SCHEMA = cv.deprecated(DOMAIN, invalidation_version="0.119")`, but that doesn't work for this integration, as it was still a platform under the sensor integration. 

So, I'm not sure how to proceed, on my end it just works fine.

Reply:
Just tested it, you are absolutely right. Sorry about that confusion.